manim/mobject/svg/opengl_text_mobject.py

```diff
@@ -49,14 +49,10 @@ def construct(self):
 import hashlib
 import os
 import re
-import sys
 import typing
 from contextlib import contextmanager
 from pathlib import Path
-from typing import Dict
```

Comment:
why this change? it is used in `t2c: Dict[str, str] = None`

Reply:
See https://lgtm.com/rules/1818040193/ 
https://lgtm.com/projects/g/ManimCommunity/manim/snapshot/506c4a87fbd35877151d92f507ef0a31f4b26d05/files/manim/mobject/svg/opengl_text_mobject.py?sort=name&dir=ASC&mode=heatmap#x50a2c203b0c6fa61:1

Reply:
ok!